docs/content/v0.20/release.md

```diff
@@ -0,0 +1,60 @@
+
+## Testing and Instructions
+
+The unit tests (of which we have almost none right now), integration tests (of which we have none yet), and end-to-end (E2E) tests in this repository are run in the context of a local copy of this repository and test that version of this repository --- not using any published release artifacts.
+
+The end-to-end tests include ones written in `bash`, and these are the only documentation telling a user how to use the present version of this repository. Again, these tests do not use any published artifacts from a release of this repo.
+
+We will introduce another category of tests, _release tests_. These test a given release, using the published artifacts of that release. These tests will be executables that a contributor invokes, passing the identifier of the desired release on the command line. We will create a GitHub workflow that can be explicitly triggered and will apply the release tests (in the branch that the workflow is directed at) to the latest release in that branch. We will also create a GitHub workflow that is simply told which release to test.
+
+We will maintain a document that lists releases that pass our quality bar. The latest of those is thus the latest stable release. This document is updated in `main` as quality evaluations come in.
+
+We will maintain an examples document that shows users how to exercise a few scenarios using the latest stable release.
+
+## Policy
+
+We aim for all regular releases to be working. In order to do that, we have to make test releases and test them. The widely recognized pattern for doing that is to make "release candidates" (i.e., releases for testing purposes) `1.2.3-rc0`, `1.2.3-rc1`, `1.2.3-rc2`, and so on, while trying to get to a quality release `1.2.3`. Once one of them is judged to be of passing quality, we declare it to be `1.2.3` (i.e., add another tag, `v1.2.3` in addition to the existing tag like `v1.2.3-rc3`).
+
+When reaching a new major or minor version, we make a release branch for it. That is, a branch named `release-$major.$minor`. At first this branch points to the same commit as `main`, and we make release `$major.$minor.0-rc0` from it. After that work can continue, if necessary, on this release branch while work on `main` proceeds to develop the next minor (or major) version.
+
+## Open questions
+
+What to do about a deliberately feature-incomplete release (development snapshot)? Would we be making releases identified `0.20.0-milestone.1` (with `0.20.0-milestone.1.rc.0`, `0.20.0-milestone.1.rc.1` and so on for testing) and `0.20.0-milestone.2` (also with its own test releases) before getting to full-featured rlease `0.20.0`? Or would we simply decleare releases `0.20.0` and `0.21.0` to be deliberately feature-incomplete and release `0.22.0` would be the full-featured release?
+
```

Comment:
I would prefer the latter.

Reply:
I think the latter would work better with the website automation that we have. It codifies a concept of release branch, at least as far as website matters are concerned.